front/app/services/stats.ts

```diff
@@ -75,24 +75,6 @@ export function ideasByStatusStream(streamParams: IStreamParams | null = null) {
 }
 export const ideasByStatusXlsxEndpoint = `${apiEndpoint}/ideas_by_status_as_xlsx`;
 
-export function ideasByTimeStream(streamParams: IStreamParams | null = null) {
```

Comment:
Awesome! We should make a note about removing the corresponding endpoints too

Reply:
yes, going to do that on a separate PR